src/layers/MultiscaleImageLayer/MultiscaleImageLayer.js

```diff
@@ -128,7 +128,7 @@ export default class MultiscaleImageLayer extends CompositeLayer {
       tileSize,
       onClick,
       extent: [0, 0, width, height],
-      minZoom: -(numLevels - 1),
+      minZoom: Math.round(-(numLevels - 1) + Math.log2(512 / tileSize)),
```

Comment:
can you add a comment? what does this expression evaluate to (or relate to the minZom level)? Why 512?